docs/source/gallery/predictive_checks/99_plot_forest_pp_obs.py

```diff
@@ -9,33 +9,34 @@
 API Documentation: {func}`~arviz_plots.plot_forest`
 :::
 """
-from importlib import import_module
-
 from arviz_base import load_arviz_data
 
 import arviz_plots as azp
 
 azp.style.use("arviz-variat")
 
-backend="none"  # change to preferred backend
-plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
-color = plot_bknd.get_default_aes("color", 3, {})[-1]
-
 idata = load_arviz_data("non_centered_eight")
 pc = azp.plot_forest(
     idata,
     group="posterior_predictive",
     combined=True,
     labels=["obs_dim_0"],
-    backend=backend,
+    backend = "none",  # change to preferred backend
+
 )
 pc.map(
     azp.visuals.scatter_x,
     "observations",
     data=idata.observed_data.ds,
     coords={"column": "forest"},
-    color=color,
+    color="gray",
+)
+
+pc.map(
+    azp.visuals.labelled_x,
+    "xlabel",
+    coords={"column": "forest"},
+    text="Observations",
+    ignore_aes="y",  # we can omit this in matplotlib, but not bokeh, plotly
```

Comment:
Feature or bug?

Reply:
matplotlib being too permissive with kwargs. Not ignoring the `y` aes means a `y=...` kwarg gets passed downstream. As the function sets the xlabel `y=...` is generally not allowed, but matplotlib does allow any kwarg that is valid for [text](https://matplotlib.org/stable/api/text_api.html#matplotlib.text.Text) even if it will be ignored as the position is set in a different way for xlabels.

Reply:
Should we allow `ignore_aes="all"` for cases like this? I think for titles, x/ylabels, grids we'll generally want to ignore all aesthetics. The defaults in `aes_map` handle this "natively" for batteries included plots, but using `.map` outside them would probably benefit from that option, I can make it an issue

Reply:
Sounds good.

Reply:
I removed the comment because it really should not be omitted even if it works in matplotlib. In addition to calling `set_xlabel(..., y=...)` it is being called by every value of `y` available which we really don't need nor want to happen.